pkg/controller/networkpolicy/networkpolicy.go

```diff
@@ -8,19 +8,21 @@ import (
 	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
 	"github.com/acorn-io/acorn/pkg/config"
 	"github.com/acorn-io/acorn/pkg/labels"
+	"github.com/acorn-io/acorn/pkg/system"
 	"github.com/acorn-io/baaah/pkg/name"
 	"github.com/acorn-io/baaah/pkg/router"
 	"github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	apierror "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/strings/slices"
 )
 
-// NetworkPolicyForApp creates a single Kubernetes NetworkPolicy that restricts incoming network traffic
+// PolicyForApp creates a single Kubernetes NetworkPolicy that restricts incoming network traffic
 // to all pods in an app, so that they cannot be reached by pods from other projects.
-func NetworkPolicyForApp(req router.Request, resp router.Response) error {
+func PolicyForApp(req router.Request, resp router.Response) error {
```

Comment:
nit: if we're changing the name, let's trim it all the way down:
```suggestion
func ForApp(req router.Request, resp router.Response) error {
```